start_chromium_with_proxy.sh

```
@@ -0,0 +1,21 @@
+
+    mkdir -p $HOME/.pki/nssdb
+    certutil -N -d sql:$HOME/.pki/nssdb --empty-password
+else
+    echo "Database already exists."
+fi
+
+CERT_FILE="/root/.mitmproxy/mitmproxy-ca.pem"
+
+
+if [ -f "$CERT_FILE" ]; then
+    certutil -d sql:$HOME/.pki/nssdb -A -t "C,," -n mitmproxy -i /root/.mitmproxy/mitmproxy-ca.pem
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)